models/gc_tiger_lb2/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Specify the base image for the environment
+FROM mhubai/base:latest
+
+# Specify/override authors label
+LABEL authors="sil.vandeleemput@radboudumc.nl"
+
+# install required dependencies for algorithm
+RUN pip3 install --no-cache-dir torch==2.0.1+cu118 torchvision==0.15.2+cu118 -f https://download.pytorch.org/whl/torch_stable.html
+
+# Install ASAP
+RUN apt-get update \
+    && apt-get -y install curl libpython3.8-dev \
+    && curl --remote-name --location "https://github.com/computationalpathologygroup/ASAP/releases/download/ASAP-2.1/ASAP-2.1-py38-Ubuntu2004.deb" \
+    && dpkg --install ASAP-2.1-py38-Ubuntu2004.deb || true \
+    && apt-get -f install --fix-missing --fix-broken --assume-yes \
+    && ldconfig -v \
+    && apt-get clean \
+    && echo "/opt/ASAP/bin" > /usr/local/lib/python3.8/dist-packages/asap.pth \
```

Comment:
Looks a bit hacky. Is this the official / stable install?

Reply:
Yes, this code came with the Dockerfile used for the Tiger challenge.

Reply:
Note, that if the Ubuntu or the Python version changes, this model will become deprecated or require a revision.

Do you know if ASAP can run on Ubuntu 22.04 or with any other Python version?
Did you think about a maintenance strategy?

Reply:
Hi, yes as discussed: There is a newer version of ASAP for Ubuntu 22.04, so once the base image moves, we can move along. The maintenance strategy is to support these Ubuntu and Python versions (potentially in a pyenv), once 22.04 is rolled out we can update accordingly. For steps beyond that we have to discuss...

Reply:
Ok, it is now part of a pipenv.